TensorFlow r1.8
Bazel: 0.14.0
Platform: Oracle Enterprise Linux 6 (derivative of RedHat Enterprise Linux 6): Linux 2.6.32-431.29.2.el6.x86_64
Bazel Command: bazel build --linkopt='-lrt' --config=mkl -c opt --copt=-mavx --copt=-mfma --copt=-msse4.1 --copt=-msse4.2 //tensorflow/tools/pip_package:build_pip_package

I finally did find a way to fix it by applying this patch:

 --- tensorflow/tensorflow.bzl.orig     +0100
 +++ tensorflow/tensorflow.bzl  +0100
 @@ -920,6 +920,7 @@
    args += [src.path]
    outputs = [ctx.outputs.cc_out, ctx.outputs.py_out]
    ctx.action(
 +      use_default_shell_env = True,
        executable=ctx.executable._swig,
        arguments=args,
        inputs=list(inputs),
which I got from here:
#4053 : https://github.com/bazelbuild/bazel/issues/4053

Note that this same issue was raised in the TensorFlow projejct:
tensorflow/tensorflow#14211 : https://github.com/tensorflow/tensorflow/issues/14211

and deemed "not a bug", kicking the issue back in a very un-Google sort of way.



pkg: pcre

pcre-devel-8.32-15.1.alios7.x86_64
pcre-8.32-15.1.alios7.x86_64
